src/code/models/link-manager.coffee

```diff
@@ -30,17 +30,17 @@ class LinkManager
     log.info("adding selection listener #{listener}")
     @selectionListeners.push listener
 
-  getLinks: () ->
-    return (value for key, value of @linkKeys)
```

Comment:
The link-manager.coffee file is the one existing coffee file I didn't finish cleaning up.  I noticed it when I was reviewing the pull request but I went ahead and pulled the trigger.  I'll clean it up when I add a coffeelint gulp task.